### PR TITLE
Update entry tutorial

### PR DESCRIPTION
**Pull request by Jacalz**
This changes the tutorial to show how to make the entry numerical instead, and fixes a few errors that were in the old tutorial.
NOTE: I could unfortunately not test run this locally as I'm on a cellular connection and couldn't quite get jekyll set up on my laptop. However, it should work (famous last words) ;)

### CODE REVIEW

File: tutorial/numerical-entry.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+layout: page
+title: Numerical Entry
+
+redirect_from: /develop/numerical-entry.html
```
Comment (andydotxyz) [inline tutorial/numerical-entry.go:5]: This is a compatibility line, it should probably be from /tutorial/keypress-on-entry.html so people will get this page instead?

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Looks great, just the redirect to old page locations that should be fixed I think.

File: tutorial/numerical-entry.go (hunk 1 above)
Comment (Jacalz) [reply tutorial/numerical-entry.go:5]: Good point. I will change it back.

File: tutorial/numerical-entry.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
```
Comment (andydotxyz) [inline tutorial/numerical-entry.go:1]: This file is `.go` it needs to be `.md` to work

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Oops, one last problem (I tested it in jekyll for you)

File: tutorial/numerical-entry.go (hunk 2 above)
Comment (Jacalz) [reply tutorial/numerical-entry.go]: Wow. That is an embarrassing mistake. Sorry. Fixed now.

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**